data-structures-queues/is-empty.js

```diff
@@ -0,0 +1,9 @@
+/* exported isEmpty */
+
+function isEmpty(queue) {
+  if (queue.peek() === undefined) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
When you have an `if` statement that returns `true` else `false` you can always just return the result of the expression.
```JAVASCRIPT
return queue.peek() === undefined
```